r/LICENSE.md

```diff
@@ -0,0 +1,194 @@
+Apache License
```

Comment:
Does this license file need to be duplicated? Can the one from the top level be used?